Src/lib/drivers/button/button.c

```diff
@@ -58,10 +59,10 @@ button_err_t button_unreg_callback(void)
 
 void button_run(void)
 {
-    if (pressed_time_ms >= timeout)
+    if (pressed_time_ms >= timeout )
     {
         objS_buttonCb(e_event_timeout);
-        pressed_time_ms = 0;
```

Comment:
Try to Avoid code commenting in pull requests

Reply:
Done